safedelete/utils.py

```diff
@@ -16,3 +19,19 @@ def can_hard_delete(obj):
     collector = Collector(using=router.db_for_write(obj))
     collector.collect([obj])
     return not bool(collector.fast_deletes)
+
+
+def related_objects(obj):
+    """ Return a generator to the objects that would be deleted if we delete "obj" (excluding obj) """
+
+    collector = NestedObjects(using=router.db_for_write(obj))
```

Comment:
I'm not sure we should get all the nested related objects there. We should only get the directly related ones, which should also cause the deletion of their related ones if they have `SOFT_DELETE_CASCADE` as well.